.eslintrc.js

```diff
@@ -71,7 +71,8 @@ module.exports = {
 					};
 					return acc;
 				}, {})),
-				extends: "extends"
+				extends: "extends",
+				constructor: "constructor"
```

Comment:
`@class` is preferred on `@constructor`, `@class` is restricted https://github.com/webpack/webpack/blob/master/.eslintrc.js#L43 =) Suggestion: use only `eslint-jsdoc-plugin` rules, we will have one state of truth